packages/app/cypress/e2e/runner/reporter.errors.cy.ts

```diff
@@ -180,6 +180,16 @@ describe('errors ui', {
       ],
     })
 
+    verify('spec unhandled rejection with string content', {
+      uncaught: true,
+      column: 20,
+      originalMessage: 'Unhandled promise rejection with string content from the spec',
+      message: [
+        'The following error originated from your test code',
+        'It was caused by an unhandled promise rejection',
+      ],
```

Comment:
```suggestion
      ],
      stackRegex: /.*/,
      hasCodeFrame: false,
```

@geritol Adding this configuration should fix the failure.

Reply:
@chrisbreiding works 🙌